stdlib/2and3/sre_compile.pyi

```diff
@@ -5,6 +5,20 @@ import sys
 from sre_parse import SubPattern
 from typing import Any, List, Pattern, Tuple, Type, Union
 
+# SRE_* constants are undocumented
```

Comment:
The implementation re-exports from sre_constants, maybe we should match

Reply:
Good catch, changed it accordingly.